components/rmrk/Gallery/GalleryItemCarousel.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div>
+    <div class="my-5" v-if="nfts.length >= MIN_NFTS">
+      <p class="subtitle is-size-4">{{ $t(`nft.${type}`) }}</p>
+      <CarouselCardList :nfts="nfts" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Provide } from 'nuxt-property-decorator'
+import { formatNFT } from '~/utils/carousel'
+import { visitedNFT } from '~/utils/localStorage'
+
+import collectionEntityById from '@/queries/rmrk/subsquid/collectionEntityById.graphql'
+import nftEntitiesByIDs from '@/queries/rmrk/subsquid/nftEntitiesByIDs.graphql'
+
+import { CarouselNFT } from '@/components/base/types'
+
+@Component({
+  components: {
+    CarouselCardList: () => import('@/components/base/CarouselCardList.vue'),
+  },
+})
+/**
+ * class name
+ */
+export default class GalleryItemCarousel extends Vue {
+  @Prop({ type: String, required: true }) type!: 'related' | 'visited'
+  @Prop({ default: '' }) readonly collectionId!: string
+  @Provide() MIN_NFTS = 3
```

Comment:
`utils/constants.ts` cries in the corner

Reply:
aahh, I see. will move that

Reply:
updated